lib/sandbox/execute.js

```diff
@@ -19,6 +19,7 @@ const _ = require('lodash'),
     EXECUTION_COOKIES_EVENT_BASE = 'execution.cookies.',
     EXECUTION_ASSERTION_EVENT = 'execution.assertion',
     EXECUTION_ASSERTION_EVENT_BASE = 'execution.assertion.',
+    EXECUTION_SKIP_EVENT = 'execution.skip.',
```

Comment:
Let's call this `execution.skipRequest.` . We might need `execution.skip.` in future where the complete execution needs to be skipped and not just a single request.

Reply:
Changed